packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -112,6 +114,7 @@ class _AutocompleteField extends StatelessWidget {
     required this.focusNode,
     required this.textEditingController,
     required this.onFieldSubmitted,
+    required this.fieldKey,
```

Comment:
Maybe just call this `key`, since it is the key of the topmost widget returned from `build`.

Reply:
Better not to not use a member of StatelessWidget that is already defined

Reply:
Yeah you're right, it would be weird to take key and not pass it to super.